src/main/scala/chiseltest/legacy/backends/verilator/VerilatorCppHarnessGenerator.scala

```diff
@@ -40,10 +47,15 @@ object VerilatorCppHarnessGenerator {
     val dutName = dut.name
     val dutApiClassName = dutName + "_api_t"
     val dutVerilatorClassName = "V" + dutName
-    // example version string: Verilator 4.038 2020-07-11 rev v4.038
-    // this will probably break when Verilator hits version 10+, but hopefully we can drop compatibility by then
-    val verilatorVersion = "verilator --version".!!.split(' ')(1).stripLineEnd
-    val verilatorRunFlushCallback = if (verilatorVersion >= "4.038") {
+    val (verilatorMajor, verilatorMinor) = verilatorVersion
+
+    val coverageInit =
+      if (verilatorMajor >= 4 && verilatorMinor >= 202)
+        """|Verilated::defaultContextp()->coveragep()->forcePerInstance(true);
```

Comment:
it's awesome that you found that!

Reply:
It's on `4.202` release notes with (literally) your name on it! :D